buddyboss/buddyboss-platform
# PROD-864 PROD-863 : invite admins of parent group into subgroup

### Changes proposed in this Pull Request:
https://buddyboss.atlassian.net/browse/PROD-863
https://buddyboss.atlassian.net/browse/PROD-864

### How to test the changes in this Pull Request:

1. Enable these settings https://i.ibb.co/hd9mnj8/hierarchies.jpg in BuddyBoss > Settings > Group.
2. Create a parent group and add some organizers, moderators and members
3. Now create a sub-group.
4. Go to frontend of the sub-group and navigate to the Send Invite tab
5. Notice that now all the members, as well as moderator and organizer of the parent group, can be invited.

### Proof Screenshots or Video


https://www.loom.com/share/ad616df285194347bcd6972ee76518c4
